sw/device/tests/chip_power_idle_load.c

```diff
@@ -0,0 +1,406 @@
+// Copyright lowRISC contributors.
```

Comment:
you can run `bazel run //quality:clang_format_fix` in your local area.
This will automatically invoke the version of `clang_format` used by CI to format your code. Otherwise this will likely always fail CI checks.

Reply:
Will run it for the final PR.